site/models/meet-events.php

```diff
@@ -28,8 +27,7 @@ public function children(): Pages
 				'content'  => [
 					'title'    => $event['name'],
 					'link'     => $event['entity_metadata']['location'],
-					'date'     => Date::optional($event['scheduled_start_time'])->setTimezone(new DateTimeZone($timezone))->format('Y-m-d H:i:s'),
-					'timezone' => $timezone
+					'date'     => Date::optional($event['scheduled_start_time'])->setTimezone(new DateTimeZone('UCT'))->format('Y-m-d\TH:i:s\Z'),
```

Comment:
`Typo fix` & `Suggestion`:

```suggestion
					'date'     => Date::optional($event['scheduled_start_time'])->setTimezone(new DateTimeZone('UTC'))->format('Y-m-d\TH:i:sp'),
```

However I think it would be even better to use a raw machine-parsable date format here and format it in the snippet (see below).